packages/andromeda-non-fungible-tokens/src/marketplace.rs

```diff
@@ -46,13 +46,13 @@ pub enum ExecuteMsg {
         token_id: String,
         token_address: String,
     },
-    /// Restricted to owner
+    #[attrs(nonpayable, restricted)]
```

Comment:
Did we remove any relevant calls to `nonpayable` from the handlers for these messages?

Reply:
No I don't think `execute_authorize_contract` and `execute_deauthorize_contract` had `nonpayable` in the first place.